addon/services/tour.js

```diff
@@ -156,6 +157,13 @@ export default Service.extend(Evented, {
   requiredElements: [],
   steps: [],
 
+  init() {
+    this._super(...arguments);
+    let owner = getOwner(this);
+    const fastboot = owner.lookup('service:fastboot');
+    this.fastboot = fastboot && fastboot.isFastBoot;
```

Comment:
Could we make this `this._isFastBoot`? Since fastboot is a service, `this.fastboot` could potentially point to the service, if we ever were to inject it. Just feels safer to set it to a different name.